### PR TITLE
Update BetterModelClassDiagram.puml

### LINKED ISSUE #97: UML for member
Create uml diagram for member class to update

### PR DESCRIPTION
**Pull request by yuvrajaryan**
Fix #97

### CODE REVIEW

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 1]
```diff
@@ -5,16 +5,19 @@ skinparam arrowColor MODEL_COLOR
...
 AddressBook *-right-> "1" UniqueMemberList
-AddressBook *-right-> "1" UniqueTagList
-UniqueTagList -[hidden]down- UniqueMemberList
-UniqueTagList -[hidden]down- UniqueMemberList
 
-UniqueTagList -right-> "*" Tag
-UniqueMemberList -right-> Member
 
-Member -up-> "*" Tag
+UniqueMemberList -right-> "*" Member
```
Comment (QinHaichen12) [inline docs/diagrams/BetterModelClassDiagram.puml:10]: aggregation relationship?

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 2]
```diff
@@ -5,16 +5,19 @@ skinparam arrowColor MODEL_COLOR
...
 
-Member *--> Name
-Member *--> Telegram
-Member *--> Room
+
+Member *-up-> "*" Tag
```
Comment (QinHaichen12) [inline docs/diagrams/BetterModelClassDiagram.puml:13]: does this mean member can contain multiple tags?

**Review by QinHaichen12 — COMMENTED** [review]
Left some comments on the diagram!

File: docs/diagrams/BetterModelClassDiagram.puml (hunk 1 above)
Comment (yuvrajaryan) [reply docs/diagrams/BetterModelClassDiagram.puml:10]: Aggregation is not recommended as per the textbook

File: docs/diagrams/BetterModelClassDiagram.puml (hunk 2 above)
Comment (yuvrajaryan) [reply docs/diagrams/BetterModelClassDiagram.puml:13]: yes

**Review by starchypotatocode — APPROVED** [review]
LGTM

**Pull request merged**